Source/DafnyCore/Options/DafnyProject.cs

```diff
@@ -17,8 +17,8 @@
 namespace Microsoft.Dafny;
 
 public class DafnyProject : IEquatable<DafnyProject> {
-  public static Option<FileInfo> FindProjectOption = new("--find-project",
-    "Given a filesystem path, search for a project file by traversing up the file tree.");
+  public static Option<bool> FindProjectOption = new("--find-project",
```

Comment:
It does not look like a hidden option, any concerns about backward compatibility?